pkg/cli/common.go

```diff
@@ -227,6 +228,7 @@ func GetBudFlags(flags *BudResults) pflag.FlagSet {
 		panic(fmt.Sprintf("error marking the pull-never flag as hidden: %v", err))
 	}
 	fs.BoolVarP(&flags.Quiet, "quiet", "q", false, "refrain from announcing build instructions and image read/write progress")
+	fs.BoolVar(&flags.IdentityLabel, "identity-label", true, "add default builder label (default true)")
```

Comment:
```suggestion
	fs.BoolVar(&flags.IdentityLabel, "identity-label", true, "add default identity label (default true)")
```
At least to be consistent with the man page